server/routes/induction/create/hopingToWorkOnReleaseCreateController.test.ts

```diff
@@ -33,24 +29,18 @@ describe('hopingToWorkOnReleaseCreateController', () => {
   }
   const next = jest.fn()
 
-  const noErrors: Array<Record<string, string>> = []
-
   beforeEach(() => {
     jest.resetAllMocks()
-    req.session = {} as SessionData
+    req.session = { prisonerSummary } as SessionData
     req.body = {}
     req.user = {} as Express.User
-    req.params = {} as Record<string, string>
-    req.params.prisonNumber = prisonNumber
+    req.params = { prisonNumber }
     req.path = `/prisoners/${prisonNumber}/induction/hoping-to-work-on-release`
   })
 
   describe('getHopingToWorkOnReleaseView', () => {
     it('should get the Hoping To Work On Release view given there is not a HopingToWorkOnReleaseForm already on the session', async () => {
       // Given
-      const prisonerSummary = aValidPrisonerSummary()
```

Comment:
I've done this same refactoring in all of the controller tests I've touched in this PR (quite a lot of them 😬 ) to reduce some of this duplication re: setup.
I've moved it up to the beforeEach hook